pipelines/GERPonHAL/gerp_on_hal.nf

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env nextflow
 
+genomes_file = file(params.genomes_file)
```

Comment:
Would it be possible to name this parameter `keep_genomes_file` to match the parameter in the `hal_to_maf.py` script?